config/initializers/date_formats.rb

```diff
@@ -0,0 +1,41 @@
+Time::DATE_FORMATS[:govuk_date] = "%-d %B %Y"
+Time::DATE_FORMATS[:govuk_date_short_month] = "%-d %b %Y"
+Date::DATE_FORMATS[:govuk_date] = "%-d %B %Y"
+Date::DATE_FORMATS[:govuk_date_short_month] = "%-d %b %Y"
+
+Time::DATE_FORMATS[:month_and_year] = "%B %Y"
+Date::DATE_FORMATS[:month_and_year] = "%B %Y"
+
+Time::DATE_FORMATS[:short_month_and_year] = "%b %Y"
+Date::DATE_FORMATS[:short_month_and_year] = "%b %Y"
+
+Time::DATE_FORMATS[:day_and_month] = "%-d %B"
+Date::DATE_FORMATS[:day_and_month] = "%-d %B"
+
+Time::DATE_FORMATS[:govuk_date_and_time] = lambda do |time|
```

Comment:
What's the benefit of putting this in an initializer rather than a module?

Reply:
🤷🏼‍♂️ I just copied this from https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/initializers/date_formats.rb – BUSTED! I’ve no idea what might be considered best practice, but it works. 😊

Reply:
Would recommend copy and pasting all of it then :stuck_out_tongue: 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/spec/config/initializers/date_formats_spec.rb

Reply:
For reference the downside of this is changes to this code will require a server restart to see in dev. Not super important.

Reply:
God damn it, I looked for a spec, but obviously not hard enough 😄